### PR TITLE
Mobile menu made with Abdullah

### PR DESCRIPTION
**Pull request by Nedramrez**
- Add a hidden mobile menu
- Add javascript code for mobile menu
- Ask to commit the coding partner

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -9,18 +9,30 @@
...
+          <div class="mobile-menu" id="mobMenu">
+            <div class="mobile-menu-items">
+              <div class="mobMenu-close">
+                <a onclick="mobileMenu()">
+                <img class="close-icon" src="assets/close-icon.svg" alt="close-icon">
+                </a>
+              </div>
+              <a onclick="mobileMenu()" href="#portfolio">Portfolio</a>
+              <a onclick="mobileMenu()" href="#about">About</a>
+              <a onclick="mobileMenu()" href="#contact">Contact</a>
```
Comment (youmari) [inline index.html:29]: - [x] please make sure that your navigation  links are clear and looks white as in the Figma design 
- Figma
image
- Yours
image

**Review by youmari — CHANGES_REQUESTED** [review]
Hi @Nedramrez , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights
✔️ Great title and short summary
✔️ Github flow 
✔️ No linter error

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: styles.css [hunk 2]
```diff
@@ -88,12 +90,53 @@ body {
...
+  backdrop-filter: blur(13px);
+  mix-blend-mode: multiply;
+  display: flex;
+  flex-direction: column;
+  height: 100vh;
+  top: 0;
+  background-color: #6070ff;
+  position: fixed;
+  width: 100%;
+}
```
Comment (vechicin) [inline styles.css]: While you did address the changes required by the previous reviewer, you didn't do so properly. Please take a closer look at the template design and make sure your design looks identical to it. Thinks to think about (but not exclusively): margins, padding, color, opacity, font, and font size.

Additionally, your mobile menu is not covering the screen vertically. That makes for bad UI/UX.

Screen Shot 2022-06-14 at 14 03 24

Please fix it.

**Review by vechicin — CHANGES_REQUESTED** [review]
# Hi @Nedramrez ✋, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet. 🚨 


 ## Highlights ⚡ 

- [x] Good job trying to implement the previous reviewer's requirements

 ## Required Changes ♻️

- [ ] I'm experiencing a slight horizontal scroll in both the mobile and desktop versions. This makes for bad UI/UX. Please fix it. I am using Google Chrome, for reference.

_Check the comments under the review._

## Optional suggestions 📂 

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by roshan-bajgain — APPROVED** [review]
Hi @Nedramrez, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- No linter errors. ✔️
- GitHub flow is followed. ✔️
- Professional README file. ✔️
- PR has a good title and summary. ✔️

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by tufoinnkuo10 — COMMENTED** [review]
Hi @Nedramrez , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request 🔕 ⛔ 🔴 

You have submitted a project, that has already been reviewed. kindly create a new pull request with new changes and ask for a review again. 😄 

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Pull request merged**